src/main/kotlin/telegram/Handling.kt

```diff
@@ -1179,23 +1182,40 @@ object MafiaHandler {
                 }
             }
             parametrized(nightCommand) {
-                try {
-                    bot.deleteMessage(ChatId.fromId(chatId), long(0))
-                    accounts.update(chatId) {
-                        menuMessageId = -1L
-                    }
-                    deleteGameTimers(bot, game.id)
-                    towns[game.id]?.let { town ->
-                        bot.sendMessage(
+                towns.get(game.id)?.let { town ->
+                    if (town.day == 1 && getHideRolesMode(game)) {
+                        val res = bot.sendMessage(
                             ChatId.fromId(chatId),
-                            "Результат дня:\n${shortLog(town).ifBlank { "Не произошло никаких событий" }}"
+                            "Выключите скрытие ролей в меню настроек"
                         )
-                        town.updateTeams()
-                        town.prepNight()
-                        showNightRoleMenu(town, chatId, bot, -1L)
+                        if (res.isSuccess) {
```

Comment:
There is a way to send simple closeable "notification" messages. Message deletion is only for specific cases. Unnecessary here.